src/components/pages/HomePage.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
- [ OPTIONAL] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.